fava/application.py

```diff
@@ -214,10 +214,20 @@ def should_collapse_account(account_name):
 
         return account_name in collapse_accounts
 
+    def sidebar_queries():
```

Comment:
If truncating can be done in Jinja (as @aumayr mentioned), then it's probably better indeed to move it to the template. Currently it became an unreadable mess...